File: code/1013/1013.cpp
#include <cmath>
#include <iostream>
using namespace std;

int getBigger(int a, int b) { return (a + b + abs(a - b)) / 2; }

int main() {

  int A, B, C;

  cin >> A >> B >> C;

  cout << getBigger(getBigger(A, B), C) << " eh o maior" << endl;

  return 0;
}

File: code/1013/1013.js
const stringValues = require("fs")
  .readFileSync("/dev/stdin", "utf8")
  .split(/\n|\s/);

const bigger = (a, b) => (a + b + Math.abs(a - b)) / 2;

const searchBigger = (values = []) =>
  values.reduce((acc, value) => bigger(acc, parseInt(value || 0)), 0);

console.log(`${searchBigger(stringValues)} eh o maior`);
